odoo/addons/test_pylint/tests/test_pylint.py

```diff
@@ -27,8 +27,12 @@ class TestPyLint(TransactionCase):
         'W0101',  # unreachable code
 
         'mixed-indentation',
+        'deprecated-module',
     ]
 
+    BAD_MODULES = [
+    ] + list(tools.SUPPORTED_DEBUGGER)
```

Comment:
I did the changes suggested by @xmo-odoo , that way they will stay synced.
Also, I kept BAD_MODULES to ease the forward port.